engine/src/mblhandlers.cpp

```diff
@@ -1336,6 +1336,31 @@ Exec_stat MCHandleSensorAvailable(void *p_context, MCParameter *p_parameters)
 	return ES_ERROR;
 }
 
+Exec_stat MCHandleAllowBackgroundLocationUpdates(void *p_context, MCParameter *p_parameters)
+{
+    MCExecContext ctxt(nil, nil, nil);
+    ctxt . SetTheResultToEmpty();
+    
+    bool t_allow_background_location_updates = false;
+    
+    if (p_parameters)
+    {
+        MCAutoValueRef t_value;
+        MCAutoBooleanRef t_bool;
+        p_parameters->eval_argument(ctxt, &t_value);
+        // PM-2015-03-11: [[ Bug 14855 ]] Evaluate correctly the second param
```

Comment:
I don't think this comment belongs here (copy & pasted?)

Reply:
Oops  :)